testdata/project-v3-multigroup/Makefile

```diff
@@ -2,7 +2,7 @@
 # Image URL to use all building/pushing image targets
 IMG ?= controller:latest
 # Produce CRDs that work back to Kubernetes 1.11 (no version conversion)
-CRD_OPTIONS ?= "crd:trivialVersions=true"
+CRD_OPTIONS ?= "crd:trivialVersions=true,preserveUnknownFields=false"
```

Comment:
Could we update the doc https://github.com/kubernetes-sigs/kubebuilder/blob/master/docs/book/src/reference/makefile-helpers.md#to-change-the-version-of-crds with these changes as well?

Reply:
I'll make a follow up for this and documenting webhook version changes.

Reply:
IMO I docs ought to do at the same pr. if we need to revert for example it will be all together.